lib/kb_Msuite/Utils/simple_run_checkm.py

```diff
@@ -0,0 +1,24 @@
+"""
+Run checkm's lineage_wf command.
+"""
+from subprocess import Popen, PIPE, STDOUT
+
+
+def run_checkm(input_dir, output_dir, log_path, options={}):
+    """
+    Run the lineage_wf command and return the stdout.
+    Arguments:
+      input_dir - required - directory of the input bins
+      output_dir - required - directory for the output
+      log_path - required - file-path for the output log
+      options - optional - dictionary of lineage_wf options
+    """
+    args = ['checkm', 'lineage_wf', input_dir, output_dir]
+    for opt, val in (options or {}).iteritems():
```

Comment:
should we check legitimacy of options? Or stdout already gives pretty informative messages for bad options?

Reply:
👍